### PR TITLE
Fix and improve featured sales agent banner component

### PR DESCRIPTION
**Pull request by RemcoSimonides**
- [x] Reinstert html code lost during media migration
- [x] Apply organization with movie query in [issue #3207](https://github.com/blockframes/blockframes/pull/3297) 
- [x] Remove unused code

### CODE REVIEW

File: libs/organization/src/lib/organization/+state/organization.model.ts [hunk 1]
```diff
@@ -20,9 +20,7 @@ export {
 
 export type AppStatus = 'none' | 'requested' | 'accepted';
 
-export interface Organization extends OrganizationDocumentWithDates {
-  movies?: Movie[];
-}
+export interface Organization extends OrganizationDocumentWithDates {}
```
Comment (RemcoSimonides) [inline libs/organization/src/lib/organization/+state/organization.model.ts:23]: I could remove this line but then we don't have a simple Organization interface

File: apps/festival/festival/src/app/marketplace/home/home.component.html [hunk 2]
```diff
@@ -66,6 +66,12 @@ <h1>Featured films</h1>
               </movie-slide>
             </bf-slide>
           </bf-slider>
+          <!-- Featured Sales Agent Banner -->
+          <ng-container *ngIf="featuredOrg$ | async as org">
+            <ng-container *ngIf="org | orgMovies:0:3 | async as movies">
+              <org-banner [org]="org" [movies]="movies"></org-banner>
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/marketplace/home/home.component.html:72]: you can do even better by removing the [movies] input and do the pipe "orgMovie" directly in the component.

File: apps/festival/festival/src/app/marketplace/home/home.component.ts [hunk 3]
```diff
@@ -69,7 +69,7 @@ export class HomeComponent implements OnInit, OnDestroy {
     ];
 
     this.orgs$ = this.organizationService
-      .queryWithMovies(ref => ref
+      .valueChanges(ref => ref
```
Comment (RemcoSimonides) [inline apps/festival/festival/src/app/marketplace/home/home.component.ts:72]: I guess Akita works like this? I am not familiar with it yet

File: libs/organization/src/lib/organization/+state/organization.model.ts (hunk 1 above)
Comment (GrandSchtroumpf) [reply libs/organization/src/lib/organization/+state/organization.model.ts:23]: keep it like that it's fine.

**Comment by RemcoSimonides** [comment]
Moved changes to already existing implement featured sales agent banner branch (https://github.com/blockframes/blockframes/pull/3303)

**Pull request closed without merging**